packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -1275,12 +1275,17 @@ export class IrisGrid extends Component<IrisGridProps, IrisGridState> {
       config: UIRollupConfig | undefined,
       aggregationSettings: AggregationSettings
     ): UITotalsTableConfig | null => {
-      const { aggregations, showOnTop } = aggregationSettings;
-      // If we've got rollups, then aggregations are applied as part of that...
-      if (
-        (config?.columns?.length ?? 0) > 0 ||
-        (aggregations?.length ?? 0) === 0
-      ) {
+      if ((config?.columns?.length ?? 0) > 0) {
+        // If we've got rollups, then aggregations are applied as part of that...
+        return null;
+      }
+
+      // Filter out aggregations without any columns actually selected
+      const aggregations = aggregationSettings.aggregations.filter(
+        agg => agg.selected.length > 0 || agg.invert
```

Comment:
If you somehow got into a state with all "selected", but `invert = true` then it's actually none selected. Not sure if that's possible though

I think `invert` makes logic around this more complex than it needs to be, but not worth changing in this PR. Like why track selected + invert instead of just selected. The invert seems to only get set on toggle all and reset. Which for reset it sets selection to none + invert. Should just set selection to all in that case

I'm guessing it's because of a "select all" case if columns or custom columns are added. Just feels like there should be `selectAll` instead or something

Reply:
We do `invert` so it scales up to large lists. If you have hundreds of columns and only deselect 1, rather than storing the hundreds of columns it just sends the one and the `invert` bit.
Not so important here, but in places like the Advanced Filter Creator it is important, as you could be deselecting values from a list of potentially millions.
In this case `invert = true` with all "selected", we flip the inversion bit (`AggregationEdit#handleSelect`). Could wrap this logic in some sort of utility method or something though to make it clearer, or resolve it to the list of aggs and just look at that. Both which I'm not inclined to do in this PR.

Reply:
Makes sense. Although the actual logic looks like invert is only false if `toggleAll` is used and thus  you can still end up with `n-1` columns in your aggregations list instead of what should be the max of `n/2` if the inversion is flipped any time it would be cheaper to track the smaller portion.

Either way not important to this change